tests/test-expected-result.c

```diff
@@ -0,0 +1,10 @@
+/*
+ * This should fail with NO-OUTPUT.
+ *
+ * @EXPECTED_RESULTS@: CORRECT, NO-OUTPUT
```

Comment:
The comment states it should fail but expected result can also be CORRECT, how does this work?

Reply:
Oh right. Artefact from my testing with problemarchive format/BAPCtools. Should just be no-output.

I won't have access to a pc the next few days though.